recipes/elector/build.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -x
+
+mkdir -p $PREFIX/bin/
+# build and install binary tools
+cd src/simulator
+sed -i "s#CC=g++#CC=${CXX}#" makefile
+make
```

Comment:
Please use @npavlovikj's suggestion `make CC="${CC}" CXX="${CXX}"` instead. It's more robust and portable than `sed` + `make`.

Reply:
You're right I think that since the variable has been defined in the makefile the change in the command line would not be taken into account.

We learn something new every day.

Thank